tests/telemetry/test_telemetry.py

```diff
@@ -128,30 +126,6 @@ def test_enable_fork_heartbeat(test_agent_session, run_python_code_in_subprocess
         assert hb["runtime_id"] == runtime_id
 
 
-def test_heartbeat_interval_configuration(run_python_code_in_subprocess):
```

Comment:
Do we need to update systems tests for this?

Reply:
Great question. I think so 😭. We have a few system tests that set this configuration to "speed up tests".

Reply:
@mabdinur did you ever figure out the answer to this question?

Reply:
Yes we will need to update these system tests: https://github.com/search?q=repo%3ADataDog%2Fsystem-tests%20DD_TELEMETRY_HEARTBEAT_INTERVAL&type=code. I don't have time to work on this so I will close this PR.